bidaf/README.md

```diff
@@ -61,17 +57,15 @@ The model has 2 outputs.
  ### Postprocessing steps
 Post processing and meaning of output
 ```
-import onnxruntime
-sess = onnxruntime.InferenceSession('bidaf.onnx')
-answer = sess.run(['start_pos', 'end_pos'], {'context_word':cw, 'context_char':cc, 'query_word':qw, 'query_char':qc})
+# assuming answer contains the np arrays for start_pos/end_pos
 start = np.asscalar(answer[0])
 end = np.asscalar(answer[1])
-print(cw[start:end+1])
+print([w.encode() for w in cw[start:end+1].reshape(-1)])
 ```
 
 For this testcase, it would output
 ```
-[['brown']].
+[b'brown'].
```

Comment:
where does the "b" come from?

Reply:
The string may contain unicode characters that cannot be print. The change to print above uses str.encode() to output the bytes array, thus b'...'

---
In reply to: [283202312](https://github.com/onnx/models/pull/159#discussion_r283202312) [](ancestors = 283202312)